### PR TITLE
Adding Kotlin dependencies.

### PR DESCRIPTION
**Pull request by KevinJCross**
### JIRA link (if applicable) ###
To help out with easing the integration test writing I'm adding the Kotlin support.

### Change description ###
Adding kotlin dependencies to enable better test writing and conscise code.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -140,6 +152,7 @@ def versions = [
 ]
 
 dependencies {
+  compile group: 'org.jetbrains.kotlin', name:'kotlin-stdlib-jdk8'
```
Comment (doncem) [inline build.gradle]: is this dependency required to be in application jar? perhaps just needed for development and can be marked as `compileOnly`

Comment (KevinJCross) [reply build.gradle]: yup .. its for the kotlin standard libraries, which includes a lot of useful extensions to java standard libraries.

Comment (timja) [reply build.gradle]: kotlin docs say use compile:
https://kotlinlang.org/docs/reference/using-gradle.html

**Comment by KevinJCross** [comment]
wow ... the "Codacy/PR Quality Review Pending" is taking a while. I wonder if its building and index or cache (21 mins and counting)?

**Pull request merged**